content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md

```diff
@@ -0,0 +1,29 @@
+Title: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar B.P!
+Date: 2016-09-16 19:00
+Category: 2016
+Slug: Announcing-the-winner-of-"Kenneth_Gonsalves_Award_2016"-Dr.Ajith_Kumar_B.P!
```

Comment:
Make sure slug is always short. Look for previous year post. https://in.pycon.org/blog/2015/kg-award-winner.html